src/main/java/me/fallenbreath/tweakermore/mixins/tweaks/features/schematicProPlace/PlacementTweaksMixin.java

```diff
@@ -131,7 +131,9 @@ private static boolean isFacingValidFor(Direction facing, ItemStack stack)
 
 		// Carpet-Extra mod accurate block placement protocol support
 		if (flexible && rotation && accurate == false &&
-				//#if MC >= 11700
+				//#if MC >= 12100
+				//$$ Configs.Generic.ACCURATE_PLACEMENT_PROTOCOL.getBooleanValue() &&
```

Comment:
It might break the compatibility with older tweakeroo version in 1.21. I'll suggest using reflection to check whether `ACCURATE_PLACEMENT_PROTOCOL` or `CARPET_ACCURATE_PLACEMENT_PROTOCOL` is the correct name (and also create a function for the config accessing). see also `ReflectionUtil#getStaticField`

Reply:
done. about masa's mod dependency, should I revert it or keep it as latest as sakura's repo?

Reply:
for stuffs in the `gradle.properties`, u can just keep the updates